src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/enchanting/AbstractEnchantmentMachine.java

```diff
@@ -55,4 +60,18 @@ protected void showEnchantmentLevelWarning(@Nonnull BlockMenu menu) {
         menu.replaceExistingItem(22, progressBar);
     }
 
+    protected boolean hasIgnoredLore(ItemStack itemStack) {
+        List<String> ignoredLore = ignoredLores.getValue();
+        if (itemStack.hasItemMeta()) {
+            List<String> itemLore = itemStack.getItemMeta().getLore();
+            if (itemLore != null && ignoredLore != null) {
```

Comment:
`ignoredLore` is not nullable, so this null check is redundant.
Same goes for `itemLore`.